File: 프로그래머스/3/43105. 정수 삼각형/정수 삼각형.py
def solution(triangle):
    #bottom->up
    h=len(triangle)
    for i in range(h-1,0,-1):
        for j in range(0,i):
            triangle[i-1][j]+=max(triangle[i][j],triangle[i][j+1])
    answer = 0
    return triangle[0][0]

File: 프로그래머스/3/43105. 정수 삼각형/정수 삼각형.js
function solution(triangle) {
    let h=triangle.length;
    for(let i=h-1;i>0;i--){
        for(let j=0;j<i;j++){
            triangle[i-1][j]+=Math.max(triangle[i][j],triangle[i][j+1])
        }
    }
    return triangle[0][0];
    
}